common/src/autogluon/common/loaders/load_s3.py

```diff
@@ -27,47 +29,85 @@ def download(input_bucket, input_prefix, local_path):
     s3.Bucket(input_bucket).download_file(input_prefix, local_path)
 
 
-def list_bucket_prefix_s3(bucket, prefix):
-    return list_bucket_prefix_suffix_s3(bucket=bucket, prefix=prefix)
-
-
-def list_bucket_prefix_suffix_s3(bucket, prefix, suffix=None, banned_suffixes=None):
+# TODO: Expand to support arbitrary list of candidate file paths
+# TODO: add local file path support
```

Comment:
Not sure if `load_s3.py` should have this functionality. Multi-location support should be a higher level abstraction/wrapper around source-specific functionality

Reply:
updated to mention to create a more generic function if doing this